themis/_index.md

```diff
@@ -11,25 +11,67 @@ Menu: "main"
 _What Themis is and why it's awesome_
 
 
-## What is Themis
+## What Themis is
+
+Themis is a cross-platform high-level cryptographic library for mobile, web and server platforms. Themis solves 90% of typical data protection use cases that are common for most apps. 
+
+Themis provides ready-made building blocks _(["crypto-systems"](#crypto-systems))_, which simplify usage of core cryptographic security operations.
+
+
+{{< hint info >}}
+Learn more about [cryptography in Themis](/themis/crypto-theory/).
+{{< /hint>}}
+
+
+
+## Why Themis
+
+Unlike many other cryptographic libraries, Themis is a high-level, easy-to-use and hard-to-misuse library _(["boring crypto"](https://speakerdeck.com/vixentael/use-cryptography-dont-learn-it))_. 
+
+Themis is made by cryptographers, but targeted on developers, so it hides cryptographic details under the hood _("secure by design")_.
+
+Themis helps to build both simple and complex cryptographic features easily, quickly and securely. Themis allows developers to focus on the main thing: developing their applications.
+
+{{< hint info >}}
+Learn more about [security design ideas behind Themis](/themis/architecture/) and [how team maintains Themis](https://speakerdeck.com/vixentael/maintaining-cryptographic-library-for-12-languages) for last 6 years.
+{{< /hint>}}
 
-TODO:
 
 
 ## Use cases that Themis solves
 
-TODO:
+* **Encrypt stored secrets** in your apps and backend (API keys, session tokens, files).
+
+* **Encrypt sensitive data fields** before storing in database (_"application-side field-level encryption"_).
+
+* Support **searchable encryption**, data tokenisation (FPE) and data masking using Themis and [Acra](https://www.cossacklabs.com/acra/).
+
+* Exchange secrets securely (**share sensitive data** between parties, build simple chat app between patients and doctors).
 
+* Build **end-to-end encryption schemes** with centralised or decentralised architecture (encrypt data locally on one app, use it encrypted everywhere, decrypt only for authenticated user).
+
+* Maintain **real-time secure sessions** (send encrypted messages to control connected devices from your app, receive real-time sensitive data from your apps to your backend).
+
+* **Compare secrets** between parties without revealing them (zero-knowledge proof-based authentication).
+
+* **One crypto-library that fits them all**: Themis is a best fit for multi-platform apps (f.e. iOS+Android+Electron app with Node.js backend), because it provides 100% compatible API and works in a same way across all supported platforms.
```

Comment:
```suggestion
* **One cryptographic library that fits them all**:
  Themis is the best fit for multi-platform apps (e.g., iOS+Android+Electron app with Node.js backend)
  because it provides 100% compatible API and works in the same way across all supported platforms.
```
I feel that “crypto-” reads more like an qualitative adjective, with a slightly inappropriate vibe.
Also, articles, and “e.g.” is used more widely in existing text.